README.md

```diff
@@ -35,8 +35,7 @@ class App extends Component {
 
   or
 - `npm i`
-- `npm run build`
```

Comment:
You still need this step `npm run build`

Reply:
Thanks for catching that.